OVERVIEW.md

```diff
@@ -281,6 +300,30 @@ Values: A single file extension or an Array of file extensions, beginning with a
 
 `.js` files are processed by default
 
+### extract
+
+Set list of file masks from which JavaScript should be extracted.
+
+Type: `Array` or `"always"` or `"auto"` or `"never"`
+
+JavaScript extracting from files that doesn't match to `fileExtensions` or `excludeFiles`. Only `html` format supported.
+
+Special values meaning:
+- `always` - JavaScript extracts from any file
```

Comment:
`JavaScript extracts from any file` -> `extract from any file`, since you use imperative mood down the road